torch/testing/__init__.py

```diff
@@ -83,7 +82,9 @@ def make_non_contiguous(tensor):
             input = input.narrow(i, bounds, tensor.size(i))
 
     input.copy_(tensor)
-    return input
+
+    # Use .data here to hide the view relation between input and other temporary Tensors
+    return input.data
```

Comment:
why can't this one use no_grad?

Reply:
Unfortunately no because we track views even when grad mode is disabled at the moment.
This will be fixed at the same time as the bug in Function that return view of intermediary results.

Reply:
but what makes you realize you need to add it here?  Like, which call is a problem?  Is it coming from gradcheck?

Reply:
This is necessary for `gradgradcheck` when we remove the `.data` in the numerical gradient for functions for which the version bumping is not an issue.
The idea is that:
- This will be required when we fix the `.data` in gradcheck.
- It took me half an hour to find that this was the issue. And this looks like a good way to document this finding.